lib/project_types/script/messages/messages.rb

```diff
@@ -5,6 +5,9 @@ module Messages
     MESSAGES = {
       script: {
         error: {
+          deprecated_ep: "This project uses Extension point %s which has been deprecated.
+ This Script will no longer function in production.",
+          deprecated_ep_cause: "Try using a different Extension point.",
```

Comment:
Same here.

Reply:
Hm, I thought I was following the pattern here but on second glance it seems that the other error messages have both lower case. I'll change it to `extension_point`